### ISSUE TITLE
[QUESTION] Status Check for Home Assistant

### ISSUE THREAD

**Issue by dipanghosh** [Lissy93/dashy#756]
### Question

Hi,

I am trying to check the status of HomeAssistant running on a container via Dashy, but this seems to fail. I have several other services, all of which are working correctly. I have all the services running behind an NGINX reverse proxy, and they are all open to the internet via subdomains. I have tried sending GET and POST requests to the HA instance manually, that works. I have also tried the suggested debug method of trying to send a GET request via the /status-check/ endpoint f Dashy, but I get the error: cannot GET /status-check/. I should mention I have authentication set for Dashy, is that the reason for this?

In any case, status checking for home assistant is not working, though the URL is reachable and responding. Any help in this regard would be much appreciated.

Best,

### Category

Status Checking

### Please tick the boxes

- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy (check the first two digits of the version number)
- [X] You've checked that this [question hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by Lissy93** [comment]
See: **[Docs → Troubleshooting → Failing Status Checks](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#status-checks-failing)**

**Comment by dipanghosh** [comment]
Hi Alicia,

Thank you for this awesome project and the continual support. I actually tried all the steps, but dashy still was not picking HA up. I posted the thread and some hours later, the problem went away. Not sure what happened there, maybe a refresh interval issue. Probably should have waited a while before ringing you, but anyway, thank you for responding.

In any case, problem solved.

**Comment by Lissy93** [comment]
No worries, glad you got it working :)
The usual cause of this is CORS, auth or a proxy issue, but I'm not sure what could have caused it to work some time later :thinking:

**Comment by dipanghosh** [comment]
Yeah, I am a bit puzzled there myself. Everything working now. This is the best dashboard out there, no questions. Thank you for making such an awesome app with godlike documentation.
